tests/publiccloud/prepare_instance.pm

```diff
@@ -37,10 +37,11 @@ sub run {
     my $provider = $self->provider_factory();
     my %instance_args;
     $instance_args{check_connectivity} = 1;
+    $instance_args{ignore_wrong_pubkey} = 1;
     $instance_args{use_extra_disk} = {size => $additional_disk_size, type => $additional_disk_type} if ($additional_disk_size > 0);
     $args->{my_provider} = $provider;
     my $instance = $provider->create_instance(%instance_args);
-    $instance->wait_for_guestregister();
+    # $instance->wait_for_guestregister(); # call removed for refactoring see its head description
```

Comment:
I don't understand why this was removed. What is the "head description"?

Reply:
yes sorry too simplified text. referring to [this L322](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/1845b47443877b6c32a2ec5f2c2dc75ee07e6220/lib/publiccloud/instance.pm#L322)

Reply:
Ah ok, but why not just fully remove it if it is not needed?

Reply:
ok